sdks/python/setup.py

```diff
@@ -187,7 +194,29 @@ def get_portability_package_data():
   # In order to find the tree of proto packages, the directory
   # structure must exist before the call to setuptools.find_packages()
   # executes below.
+  # while True: print(os.listdir())
   generate_protos_first()
+
+  # generate cythonize extensions only if we are building a wheel or
+  # building an extension or running in editable mode.
```

Comment:
is  '-e . ' an  editable_wheel mode or a different one?

Reply:
-e is a editable installation.  During editable installations, the pep 517 creates an editable wheel.  There is no `editable_wheel` mode.